File: TheLastPlanet.Client/CONFIGURAZIONE/Lavori/Generici/Pescatore.lua
Config.Client.Lavori.Generici.Pescatore = {
    TempoPescaDinamico = false, -- se true, la pesca è più realistica.. può volerci anche 1 minuto o 2 di attesa
    TempoFisso = 10, -- secondi se il tempo di pesca dinamico è false

    PrezzoVenditaPesce = 10, -- guadagno al pesce (moltiplicato per peso e grandezza ;) )
    PrezzoVenditaAltro = 20, -- come sopra ma per altre cose.. frutti di mare.. gamberi..
    
    LuoghiVendita = {
        vector3(-3251.2, 991.5, 11.49),
        vector3(3804.0, 4443.3, 3.0),
        vector3(2517.6, 4218.0, 38.8),
        vector3(-1.827, 6488.367, 31.506),
        vector3(-1552.16, -966.732, 13.017),
        vector3(-1017.513, -1350.519, 5.473)
    },

    AffittoBarca = {
        {
            Menu = vector3(3857.235, 4459.287, 1.84),
            SpawnBarca = vector4(3856.264, 4455.604, 1.405, 268.4614), -- x, y, z, h
            CancellaBarca = vector3(0,0,0),
            Barche = {
--              "TUG", -- non sono sicuro di voler dare il tug (barcone immenso)
                "DINGHY"
            }
        },
    },

--[[
    Pesci = {
        Mare = {
            facile = {
                "orata",
                "sogliola",
                "branzino"
            },
            medio = {
                "branzino",
                "orata",
                "sgombro",
                "sogliola",
            },
            avanzato = {
                "branzino",
                "orata",
                "sgombro",
                "sogliola",
                "tonno",
                "pescespada"
            }
        }
        AcquaDolce = {
            facile = {},
            medio = {},
            avanzato = {}
        }
    }

]]
    Pesci = {
        facile = {
            "orata",
            "sogliola",
            "branzino"
        },
        medio = {
            "branzino",
            "orata",
            "sgombro",
            "sogliola",
        },
        avanzato = {
            "branzino",
            "orata",
            "sgombro",
            "sogliola",
            "tonno",
            "pescespada",
        }
    }
}

File: Impostazioni.Client/Lavori/Generici/Pescatore.cs
using System.Collections.Generic;
using CitizenFX.Core;

namespace Impostazioni.Client.Configurazione.Lavori.Generici
{
    public class Pescatori
    {
        public bool TempoPescaDinamico = true;
        public int TempoFisso = 10;
        public int PrezzoVenditaPesce = 10;
        public int PrezzoVenditaAltro = 20;
        public List<Vector3> LuoghiVendita = new()
        {
            new Vector3(-3251.2f, 991.5f, 11.49f),
            new Vector3(3804.0f, 4443.3f, 3.0f),
            new Vector3(2517.6f, 4218.0f, 38.8f),
            new Vector3(-1.827f, 6488.367f, 31.506f),
            new Vector3(-1552.16f, -966.732f, 13.017f),
            new Vector3(-1017.513f, -1350.519f, 5.473f)
        };

        public Vector4 SpawnBarca = new Vector4(3856.264f, 4455.604f, 1.405f, 268.4614f);
        public List<string> Barche = new List<string>()
        {
            //"TUG", //-- non sono sicuro di voler dare il tug (barcone immenso)
            "DINGHY"

        };
        public PesciPescati Pesci = new PesciPescati()
        {
            facile = new List<string>() { "orata", "sogliola", "branzino" },
            medio = new List<string>() { "orata", "sogliola", "branzino", "sogliola" },
            avanzato = new List<string>() { "orata", "sogliola", "branzino", "tonno", "pescespada" },
        };
    }
    
    public class PesciPescati
    {
        public List<string> facile = new List<string>();
        public List<string> medio = new List<string>();
        public List<string> avanzato = new List<string>();
    }

}